t/plugin/opentelemetry-bugfix-pb-state.t

```diff
@@ -0,0 +1,192 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+use t::APISIX 'no_plan';
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    if (!$block->extra_yaml_config) {
+        my $extra_yaml_config = <<_EOC_;
+plugins:
+    - example-plugin
+    - key-auth
+    - opentelemetry
+plugin_attr:
```

Comment:
Don't you need the grpc-transcode plugin?

Reply:
No.
https://github.com/apache/apisix/blob/71c6048e3e95c0c85936e828b87fa50c5ba87d1b/apisix/admin/proto.lua#L23
Mark my last words. It does not depend on a specific plugin. Instead, it's a general problem.